tests/schema_registry/conftest.py

```diff
@@ -164,8 +169,28 @@ def __init__(self, conf):
         adapter.register_uri('POST', self.subject_versions,
                              json=self.post_subject_version_callback)
 
+        adapter.add_matcher(self._auth_matcher)
         self._rest_client.session.mount('http://', adapter)
 
+    @classmethod
+    def _auth_matcher(cls, request):
+        headers = request._request.headers
+
+        # Pass request to downstream matchers
+        if 'Authorization' not in headers.keys():
+            return None
+
+        userinfo = headers.get('Authorization').split(" ")[1]
```

Comment:
And here's a second lookup. Just do one lookup.

Reply:
Inconsistent use of single/double quotes on the same line.

Reply:
Will throw an IndexError if the Authorization header does not contain a space, but maybe that's okay here.